src/lib/mcp.ts

```
@@ -0,0 +1,316 @@
+
+            for (const tool of result.tools) {
+                // Apply filters
+                if (options?.include && !options.include.some(pattern => tool.name.includes(pattern))) {
+                    continue;
+                }
+                if (options?.exclude && options.exclude.some(pattern => tool.name.includes(pattern))) {
+                    continue;
+                }
+
+                // Apply schema if provided
+                if (options?.schemas && options.schemas[tool.name]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)